javascript-arrays/main.js

```diff
@@ -0,0 +1,22 @@
+var colors;
+colors = ['red', 'white', 'blue'];
+console.log('value of colors[0]', colors[0]);
+console.log('value of colors[1]', colors[1]);
+console.log('value of colors[2]', colors[2]);
+
+console.log('America is', colors[0] + ', ' + colors[1] + ', ' + 'and ' + colors[2]);
+
+colors[2] = 'green';
+console.log('Mexico is', colors[0] + ', ' + colors[1] + ', ' + 'and ' + colors[2]);
+console.log('value of colors', colors);
+
+var students;
+students = ['Mac', 'Charlie', 'Dee', 'Dennis'];
+var numberOfStudents = students.length;
+console.log('There are ', numberOfStudents + ' students in the class.');
+var lastIndex = numberOfStudents - 1;
+
+lastIndex = students[3];
+var lastStudent = lastIndex;
+console.log('The last student in the array is ', lastStudent);
```

Comment:
You want to use `lastIndex` as the last index in the array, you don't need to remodify it.